lib/straw_hat_map/world/countries/country.ex

```diff
@@ -7,6 +7,7 @@ defmodule StrawHat.Map.Country do
   use StrawHat.Map.Schema
   alias StrawHat.Map.Continents
   alias StrawHat.Map.State
+  alias StrawHat.Map.Regex
```

Comment:
Most of the time you are using the multi-alias/require/import/use syntax, but here you are using multiple single directives